### PR TITLE
feat(cast): remove periods from wallet new

### PR DESCRIPTION
**Pull request by rootulp**
## Motivation

Fixes https://github.com/gakonst/foundry/issues/790

## Solution

Remove trailing periods from `cast wallet new` and `cast wallet vanity` output.

### CODE REVIEW

**Comment by rootulp** [comment]
Draft PR because I haven't tested this yet. I'm not sure how to get my local `cast wallet new` to point to the binary built by my local repo. Once I figure it out, I'll try to add to https://github.com/gakonst/foundry/blob/master/CONTRIBUTING.md

File: cli/src/cast.rs
Comment (rootulp) [inline cli/src/cast.rs:637]: I noticed `cast wallet new` is not the only command that outputs Address and Private Key. `cast wallet vanity` also outputs these fields. I removed trailing periods here too for consistency.

**Review by mattsse — APPROVED** [review]
thanks!
lgtm already

**Comment by rootulp** [comment]
Wow, [foundryup](https://github.com/gakonst/foundry/tree/master/foundryup) is a thing, super cool! Tested via:

```bash
foundryup --repo rootulp/foundry --branch rp/remove-trailing-periods
```

### Screenshot

Screen Shot 2022-02-22 at 7 35 50 PM

Commands from [Resolving an issue](https://github.com/gakonst/foundry/blob/master/CONTRIBUTING.md#resolving-an-issue)
```bash
cargo check --all # passes
cargo test --all --all-features # passes
cargo +nightly fmt -- --check # error: toolchain 'nightly-aarch64-apple-darwin' is not installed
cargo +nightly clippy --all --all-features -- -D warnings # error: toolchain 'nightly-aarch64-apple-darwin' is not installed
```

The last two errors seem non-concerning and unique to my dev environment (?). Defer to reviewers if they should block this PR.

**Comment by mattsse** [comment]
> ```shell
> cargo +nightly fmt -- --check # error: toolchain 'nightly-aarch64-apple-darwin' is not installed
> cargo +nightly clippy --all --all-features -- -D warnings # error: toolchain 'nightly-aarch64-apple-darwin' is not
> ```

This looks M1 related, perhaps `rustup update` resolves this

**Comment by rootulp** [comment]
> This looks M1 related, perhaps rustup update resolves this

Thanks for the idea! I am on an M1 MacBook Pro. I just tried `rustup update` but it didn't resolve.

**Comment by gakonst** [comment]
@rootulp Could you try doing `rustup install nightly`? Think it'll fix your issue

**Comment by rootulp** [comment]
Thanks for the tip! My M1 MacBook Pro is currently being repaired but I will try that when I get it back (< 1 week).

**Pull request merged**
